docs/templates/nextflow_dsl1/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM nfcore/base:1.13.3
+LABEL authors="your name"
+      description="Docker image containing all software requirements for the [execution_tool] pipeline"
+
+# Install the conda environment
+COPY environment.yml /
+RUN conda env create --quiet -f /environment.yml && conda clean -a
+
+# Add conda installation dir to PATH (instead of doing 'conda activate')
+ENV PATH /opt/conda/envs/[execution_workflow]/bin:$PATH
+
+# Dump the details of the installed packages to a file for posterity
+RUN conda env export --name [execution_workflow] > [execution_workflow].yml
+
+# Instruct R processes to use these empty files instead of clashing with a local version
+RUN touch .Rprofile
+RUN touch .Renviron
```

Comment:
Could be combined so Docker doesn't have to dump an intermediate.

```bash
RUN touch .Rprofile .Renviron
```

Reply:
I have merged them into one line and have mute them, so if people need R then they will unmute this line.